src/implementations/config/create.ts

```diff
@@ -22,14 +26,19 @@ export const create = async ({
   outputDir,
   configTemplatePath,
   configTemplateUrl,
+  network,
 }: CreateConfigCommand): Promise<string> => {
   const walletStr = await readFile(encryptedWalletPath);
   const { address } = JSON.parse(walletStr);
   info(`Wallet detected at ${encryptedWalletPath}`);
 
+  const passedOnWallet = await getWalletOrSigner({ network, encryptedWalletPath });
```

Comment:
can be just `wallet`?

Reply:
ok, updated

Reply:
thats cool, but the other places?

Reply:
what u mean? other places have clash of variable naming